Mathlib/Probability/Variance.lean

```diff
@@ -151,7 +172,9 @@ theorem evariance_mul (c : ℝ) (X : Ω → ℝ) (μ : Measure Ω) :
   rw [mul_comm]
   simp_rw [← smul_eq_mul, ← integral_smul_const, smul_eq_mul, mul_comm]
 
-scoped notation "eVar[" X "]" => ProbabilityTheory.evariance X MeasureTheory.MeasureSpace.volume
+lemma variance_eq_integral (hX : AEMeasurable (fun ω ↦ (‖X ω - ∫ ω', X ω' ∂μ‖₊ : ℝ≥0∞) ^ 2) μ) :
+    Var[X ; μ] = ∫ ω, (X ω - μ[X]) ^ 2 ∂μ := by
+  simp [variance, evariance, ← integral_toReal hX (by simp [← ENNReal.coe_pow])]
```

Comment:
This lemma seems out of place in this PR, which should be only about the new notations. Could you please remove it?

Reply:
I meant to ask you: do you think this lemma is worth having? The measurability condition is a bit weird

Reply:
Perhaps simply replace the measurability condition by `AEMeasurable X µ`? It seems that the only way we will ever prove `AEMeasurable (fun ω ↦ (‖X ω - ∫ ω', X ω' ∂μ‖₊ : ℝ≥0∞) ^ 2) μ` will be as a consequence of that.

Reply:
I'm now adding this lemma in #20814